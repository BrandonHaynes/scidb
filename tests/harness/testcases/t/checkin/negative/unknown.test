
--setup

--test
--error --code=scidb::SCIDB_SE_QPROC::SCIDB_LE_ARRAY_DOESNT_EXIST "scan(unknown)"

--cleanup

