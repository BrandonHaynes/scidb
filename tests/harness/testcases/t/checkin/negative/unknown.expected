[An error expected at this place for the query "scan(unknown)". And it failed with error code = scidb::SCIDB_SE_QPROC::SCIDB_LE_ARRAY_DOESNT_EXIST. Expected error code = scidb::SCIDB_SE_QPROC::SCIDB_LE_ARRAY_DOESNT_EXIST.]

